stage #1/module #1. Intro, Infrastructure & Instruments, Fundamentals/lesson #3. Classes, Intro to OOP/Weapon/src/test/java/com/epam/training/weapons/MainTest.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.training.weapons;
+
+import com.epam.training.weapons.model.Division;
+import com.epam.training.weapons.view.DivisionPrinter;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class MainTest {
```

Comment:
we don't test main, and it's an integration test, not unit now

Reply:
Will be removed